docs/content/concepts/automation/declarative-automation/customizing-automation-conditions.mdx

```diff
@@ -0,0 +1,267 @@
+      </td>
+      <td>
+        True for any upstream partition. Can be used with <code>.allow()</code>{" "}
+        and <code>.ignore()</code> to target specific upstream assets. Refer to
+        the <a href="#targeting-dependencies">Targeting dependencies</a> section
+        for an example.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <code>AutomationCondition.all_deps_match(A)</code>
+      </td>
+      <td>
+        True for at least one partition of each upstream asset. Can be used with{" "}
+        <code>.allow()</code> and <code>.ignore()</code> to target specific
+        upstream assets. Refer to the{" "}
+        <a href="#targeting-dependencies">Targeting dependencies</a> section for
+        an example.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <code>AutomationCondition.any_downstream_condition()</code>
+      </td>
+      <td>
+        Any <PyObject object="AutomationCondition" /> on a downstream asset
+        evaluates to true
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+---
+
+## Targeting dependencies
+
+Upstream assets commonly influence downstream materialization decisions. To create automation conditions that target dependencies, use the `AutomationCondition.any_deps_match()` operator. This operator takes an arbitrary <PyObject object="AutomationCondition" />, applies it to each upstream asset, and then maps the results to the corresponding downstream partitions.
+
+This operator and `AutomationCondition.all_deps_match()` can be further customized to only target specific sets of upstream assets by using `.allow()` and `.ignore()`.
+
+For example, to target updates from a specific asset group, you can use `any_deps_match` with the `newly_updated` operand and tell it to target only the `metrics` asset group:
+
+```python
+from dagster import AssetSelection, AutomationCondition
+
+AutomationCondition.any_deps_match(
+    AutomationCondition.newly_updated()
+).allow(AssetSelection.groups("metrics"))
+```
+
+Or to ignore missing partitions from an upstream asset, you can use `any_deps_match` with the `missing` operand and tell it to ignore a specific asset:
+
+```python
+AutomationCondition.any_deps_match(
+    AutomationCondition.missing()
+).ignore(AssetSelection.keys("taxi_trips"))
+```
+
+---
+
+## Describing conditions with labels
+
+When there are a large number of sub-conditions that make up an <PyObject object="AutomationCondition" />, it can be difficult to understand and troubleshoot the condition. To make conditions easier to understand, you can attach labels to sub-conditions, which will then display in the Dagster UI.
+
+Arbitrary string labels can be attached to any node in the <PyObject object="AutomationCondition" /> tree by using the `with_label()` method, allowing you to describe the purpose of a specific sub-condition. For example:
+
+```python
+from dagster import AutomationCondition
+
+any_parents_missing = AutomationCondition.any_deps_match(
```

Comment:
nit: the name of this variable should be something else (could just be "my_condition")